src/back/glsl/mod.rs

```diff
@@ -312,7 +312,9 @@ pub struct Writer<'a, W> {
     /// (generated by a [`Namer`](crate::proc::Namer))
     names: crate::FastHashMap<NameKey, String>,
     /// A map with all the names needed for reflections
-    reflection_names: crate::FastHashMap<Handle<crate::Type>, String>,
+    reflection_names_structs: crate::FastHashMap<Handle<crate::Type>, String>,
```

Comment:
Maybe `reflection_names_buffers` or `reflection_names_uniforms`?

Reply:
why do we ever need to reflect the *type* names? they shouldn't be exposed anywhere in the GL API

Reply:
We need names only for uniforms here - https://github.com/gfx-rs/wgpu/blob/master/wgpu-hal/src/gles/device.rs#L38.

> they shouldn't be exposed anywhere in the GL API

Performance? We already got this info.

Reply:
I renamed it to `reflection_names_uniforms`.

Reply:
@Gordon-F that lookup goes with a variable handle, not a type handle. I'm asking why we need to look up any reflection based on the type.